README.md

```diff
@@ -160,6 +160,22 @@ doubleAndTakeEvens(List.of(1, 2, 3))
 
 Returns **[Function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function)** 
 
+### concat
+[src/concat.js:18-18](src/concat.js#L18)
+
+Joins two `Iterable.Indexed` objects together.
+
+**Examples**
+
+```javascript
+// Arrays
+concat(List([3]), List([1, 2])); // Returns List [ 1, 2, 3 ]
+const addY = concat(List(['y']);
+addY(List(['x'])); // Returns List [ 'x', 'y' ]
+```
+
+Returns `Iterable` with the concatenated value. If an indexed `Iterable`, does not override existing values. If a keyed `Iterable`, acts like `merge`.
```

Comment:
> If a keyed `Iterable`, acts like `merge`

Is this true still? It looks like it no longer works for `Map`, `Set`, and native objects?

Should this work for `Set`? I expect most methods on `List`s to work on `Set`s